.github/workflows/CI-distro.yml

```
@@ -0,0 +1,110 @@
+    strategy:
+      fail-fast: false
+      matrix:
+        julia-version:
+          - '1'
+        julia-arch:
+          - x64
+        os:
+          - ubuntu-latest
+        gap-package: ${{fromJSON(needs.generate-matrix.outputs.gap-packages)}}
+        exclude:
+          - gap-package: 'alnuth'                     # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          #- gap-package: 'autodoc'                    # test failure
+          - gap-package: 'caratinterface'             # `Carat program Bravais_inclusions failed with error code 1`
+          - gap-package: 'curlinterface'              # no jll
+          - gap-package: 'example'                    # no jll
+          - gap-package: 'examplesforhomalg'          # `Error, found no GAP executable in PATH`
+          #- gap-package: 'gapdoc'                     # test failure
+          - gap-package: 'guarana'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'hap'                        # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'hapcryst'                   # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'io'                         # segfaults, most likely due to child process handling
+          - gap-package: 'itc'                        # dependency `xgap` has no jll
+          - gap-package: 'localizeringforhomalg'      # `Error, found no GAP executable in PATH`
+          - gap-package: 'normalizinterface'          # Tests fail: NormalizInterface currently bundles Normaliz 3.9.3 and its test suite is tuned to that, but our JLL builds it against normaliz_jll which has 3.10.2
+          #- gap-package: 'packagemanager'             # test failure
+          - gap-package: 'polenta'                    # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'polycyclic'                 # test suite currently broken and also disabled by GAP package distribution
+          - gap-package: 'polymaking'                 # `polymake command not found. Please set POLYMAKE_COMMAND by hand`
+          - gap-package: 'profiling'                  # segfaults during testing
+          - gap-package: 'radiroot'                   # `AL_EXECUTABLE, the executable for PARI/GP, has to be set`
+          - gap-package: 'ringsforhomalg'             # `Error, found no GAP executable in PATH`
+          - gap-package: 'semigroups'                 # no jll
+          #- gap-package: 'typeset'                    # needs `dot2tex` executable
+          #- gap-package: 'wpe'                        # ???
+          - gap-package: 'xgap'                       # no jll
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: "Install extra dependencies"
+        run: sudo apt-get install --no-install-recommends texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended
```

Perhaps `typeset` can be made to work by doing this:
```suggestion
        run: sudo apt-get install --no-install-recommends texlive-latex-base texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended dot2tex
```
But it might need more; in the GAP PackageDistro CI tests we also install `graphviz` and a few other things, see https://github.com/gap-system/PackageDistro/blob/35c6ee0f972a502ca916d5e7937deae557622d55/tools/gather_dependencies.py#L45